src/mock/promotion.ts

```
@@ -0,0 +1,53 @@
+  {
+    code: "EXPIRED123",
+    discount: 2000,
+    description: "만료된 코드",
+    isActive: false,
+  },
+];
+
+// 프로모션 코드 검증 함수
+export const validatePromotionCode = (code: string): PromotionCode | null => {
+  const foundCode = promotionCodes.find(
+    (promo) => promo.code.toLowerCase() === code.toLowerCase() && promo.isActive
```

ESLint

[error] Replace `·promo.code.toLowerCase()·===·code.toLowerCase()·&&·promo.isActive` with `⏎······promo.code.toLowerCase()·===·code.toLowerCase()·&&·promo.isActive,`

(prettier/prettier)